docs/advanced/jinja_env.rst

```diff
@@ -0,0 +1,16 @@
+.. _jinja-env:
+
+Customizing the Jinja2 environment
+----------------------------------------------
+
+The special template variable ``_jinja2_env_vars`` can be used
+to customize the [Jinja2 environment](https://jinja.palletsprojects.com/en/3.0.x/api/?highlight=lstrip_blocks#jinja2.Environment).
```

Comment:
The current link will highlight `lstrip_blocks` on the page, and 3.1 is the current Jinja version. Please use this instead:

```suggestion
to customize the [Jinja2 environment](https://jinja.palletsprojects.com/en/3.1.x/api/#jinja2.Environment).
```